cwp/cwp-wg-papers.md

```diff
@@ -22,16 +22,52 @@ recommended.
     - Download your Google Doc as docx, a.k.a. MS Word, format
     - Using [pandoc](https://pandoc.org/), which is easily [installed](https://pandoc.org/installing.html),
       convert your docx file to latex:
-        - `pandoc -f docx -t latex -o wgpaper.tex wgpaper.docx`
+        ```bash
+        pandoc -f docx -t latex -o wgpaper.tex wgpaper.docx
+        ```
     - The conversion is generally very accurate, but within `itemize` environments you
       may find that text is inside a spurious `quote` environment and that some
       odd non-breaking spaces appear. Just delete these.
-1. Adopt the JHEP preprint style, using the template in the
+1. Adopt the [JHEP preprint](https://jhep.sissa.it/jhep/help/JHEP/TeXclass/DOCS/JHEP-author-manual.pdf)
+   style, using the template in the
    [software development paper](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-13_soft-dev/latex/).
     - Copy the `jheppub.sty` file into your latex area as it is needed for arXiv side compilation.
-1. The paper title should be `HEP Software Foundation Community White Paper Working Group \-- *My Working Group*`.
-1. You will need to add an abstract for arXiv.
-1. Add a copyright and license statement as a comment - we strongly recommend <br>`% Copyright (C) 2018, The HSF Community White Paper authors, licence CC-BY-4.0.`
+1. Build the author list: the **strongly recommended** way is to create a text file 
```

Comment:
Give an example here - the DOMA paper would be ideal so that people have a clearer idea how this work,

Reply:
Added